src/Components/Updater/config/updater.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+use LaravelZero\Framework\Components\Updater\GithubStrategy;
+
+return [
+
+    /*
+    |--------------------------------------------------------------------------
+    | Self-updater Strategy
+    |--------------------------------------------------------------------------
+    |
+    | Here you may specify which update strategy class you wish to use when
+    | updating your application via the "self-update" command. This must
+    | be a class that implements the StrategyInterface from Humbug.
+    |
+    */
+
+    'strategy' => GithubStrategy::class,
```

Comment:
Therefore this class is no longer internal.

Also, can you create a interface ( for us ) that extends from the Humbug one?

Reply:
Ah yes, I guess not. I'm happy to make a new interface that extends the Humbug one. 👍

----

With the new interface, if we wanted to use a class like [`Humbug\SelfUpdate\Strategy\GitHubStrategy`](https://github.com/humbug/phar-updater/blob/master/src/Strategy/GithubStrategy.php), we would have to create a new class that is like, I guess that seems fine. :+1:

```php
<?php

use Humbug\SelfUpdate\Strategy\GitHubStrategy as BaseGitHubStrategy;
use LaravelZero\Framework\Components\Updater\StrategyInterface;

class GitHubStrategy extends BaseGitHubStrategy implements StrategyInterface
{
}
```

Or I guess as it is currently internal (while we are making it non-internal) we could add a directory called `Strategy` that contains the strategies in case we want any more in future?

Reply:
Sure. 👍